tool/tbot/main.go

```diff
@@ -231,6 +240,56 @@ func onConfigure(
 	return nil
 }
 
+func onMigrate(
+	cf config.CLIConf,
+	stdout io.Writer,
+) error {
+	out := stdout
+	outPath := cf.ConfigureOutput
+	if outPath != "" {
+		f, err := os.Create(outPath)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		defer f.Close()
+		out = f
+	}
+
+	if cf.ConfigPath == "" {
+		return trace.BadParameter("source config file must be provided with -c")
+	}
+
+	// We do not want to load an existing configuration file as this will cause
+	// it to be merged with the provided flags and defaults.
+	cfg, err := config.ReadConfigFromFile(cf.ConfigPath, true)
```

Comment:
Should we forbid users from re-writing the original config (cf.ConfigureOutput==cf.ConfigPath) so users can roll-back without having to migrate the config again?

Reply:
Yeah - this seems like a reasonable idea. I'll add that.

Reply:
01bd6dbc65